src/pdmat.jl

```diff
@@ -31,12 +62,8 @@ AbstractPDMat(A::Cholesky) = PDMat(A)
 
 ### Conversion
 Base.convert(::Type{PDMat{T}}, a::PDMat{T}) where {T<:Real} = a
```

Comment:
It seems this can be removed?
```suggestion
```

Reply:
Not if you want it to remain a no-op. The generic fallback `convert(::Type{T}, x::T)` defined in `Core.Compiler` is of lower specificity than the method below this one.

Reply:
It seems we have to keep the method below it. And then this one is necessary if we want it to remain a no-op; the default fallback `convert(::Type{T}, x::T) where T = x` defined in `Core.Compiler` is too low specificity to be the dispatch target, given the method below this one.